bitnami/redmine/templates/deployment.yaml

```diff
@@ -14,7 +15,7 @@ spec:
   selector:
     matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
       app.kubernetes.io/component: redmine
-  replicas: {{ .Values.replicas }}
+  replicas: {{ .Values.replicaCount }}
```

Comment:
Should we mention in the README.md that this parameter was renamed?

Reply:
Done